modules/python/package/cv2/__init__.py

```diff
@@ -39,7 +39,7 @@ def bootstrap():
     l_vars = locals()
 
     if sys.version_info[:2] < (3, 0):
-        from cv2.load_config_py2 import exec_file_wrapper
+        from cv2. load_config_py2 import exec_file_wrapper
```

Comment:
Do we need this change?

Reply:
I do not changes these. This problem occurred after my rebase.